### DISCUSSION TITLE
How to show/change content of buttons

### DISCUSSION THREAD

**Discussion by db-conception** [dargmuesli/nuxt-cookie-control#144] [Q&A]
Hello,

I am sorry I may be don't understand how to use well your plugin. But I have filled white button without text or icon inside. I don't see in your getting started how to set content of button.

https://i.imgur.com/g9QwCTo.png

Can you help me please

**Comment by dargmuesli** [comment]
Which version of this module are you using? I do not recognize the visible text left to the buttons from all existing locale files, but it seems to be French. could you please clarify?
